src/Config/SettingsTab.ts

```diff
@@ -125,6 +125,17 @@ export class SettingsTab extends PluginSettingTab {
                 });
             });
 
+        new Setting(containerEl)
```

Comment:
The new setting looks like this:

![image](https://user-images.githubusercontent.com/4840096/222954010-6438b169-a94b-4ad0-9d9b-33202ead6306.png)


It feels a bit more logical to me to put creation date before done date, in terms of the lifecycle of task...

Reply:
Change order of the components.